cilium-dbg/cmd/policy_import.go

```diff
@@ -38,12 +38,6 @@ var policyImportCmd = &cobra.Command{
 				return
 			}
 
-			for _, r := range ruleList {
-				if err := r.Sanitize(); err != nil {
-					Fatalf("%s", err)
-				}
-			}
```

Comment:
Dropping the sanitization is one option, but I wonder if it would be better to just pass a parameter that can be used to determine how to sanitize the configuration. For instance if you're trying to do some sanitization based on an agent-side setting, then maybe the client by default should not apply that specific validation check, but the server side should validate based on the configuration.

I could imagine passing an object in here that has a function to check the specific option that you want to test. For the server, it will pass an object with a function that checks the state of `pkg/option/...`. For the client, the function would return a result that provides the least strict validation.